priv/still/profiler.slime

```diff
@@ -0,0 +1,179 @@
+---
+permalink: profiler.html
+title: Still Profiler Output
+---
+
+<%
+  eerie_black = "#222222"
+  lemon_meringue = "#F3EAC3"
+  liver_chestnut = "#A46F41"
+
+  grid_size = "#{1440 - 54 - 54}px"
+%>
```

Comment:
btw, these could be declared in frontmatter, you can even use EEx in frontmatter

Reply:
I'm not passing through the frontmatter, I don't want to get this through any pipeline, to reduce overhead, since it's only a profiler report.

Reply:
What do you mean? I can see a front matter config in this page :o

Reply:
After your comment I went to remove it and somehow it wasn't committed. Neither this one nor the previous comment